### PR TITLE
Notebook Example save/load Profile

### PR DESCRIPTION
**Pull request by taylorfturner**
Update notebook example to include example of `json` save and load

### CODE REVIEW

File: examples/data_profiler_demo.ipynb [hunk 1]
```diff
@@ -407,7 +407,11 @@
    "id": "de3e0e30-33e3-4b18-8240-7e5c6029eb97",
    "metadata": {},
    "source": [
-    "Not only can the Profiler create and update profiles, it's also possible to save, load then manipulate profiles."
+    "Not only can the Profiler create and update profiles, it's also possible to save, load then manipulate profiles.\n",
+    "\n",
+    "Options for saving and loading profiles, currently supported are:\n",
+    "- Pickle\n",
+    "- JSON"
```
Comment (taylorfturner) [inline examples/data_profiler_demo.ipynb:414]:
```suggestion
    "- JSON: Only StructuredProfiler is supported by JSON
```

File: examples/data_profiler_demo.ipynb [hunk 2]
```diff
@@ -407,7 +407,11 @@
    "id": "de3e0e30-33e3-4b18-8240-7e5c6029eb97",
    "metadata": {},
    "source": [
-    "Not only can the Profiler create and update profiles, it's also possible to save, load then manipulate profiles."
+    "Not only can the Profiler create and update profiles, it's also possible to save, load then manipulate profiles.\n",
+    "\n",
+    "Options for saving and loading profiles, currently supported are:\n",
+    "- Pickle\n",
+    "- JSON: Only StructuredProfiler is supported by JSON
```
Comment (taylorfturner) [inline examples/data_profiler_demo.ipynb:414]:
```suggestion
    "- JSON: Only StructuredProfiler is supported by JSON\n"
```

**Comment by taylorfturner** [comment]
Pending #925

**Review by micdavis — CHANGES_REQUESTED** [review]
Needs to be run once the load functionality is in first.

**Review by micdavis — APPROVED** [review]

File: examples/data_profiler_demo.ipynb [hunk 3]
```diff
@@ -435,6 +439,31 @@
...
+   "id": "8e713fc2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load data\n",
+    "data = dp.Data(os.path.join(data_folder, \"csv/diamonds.csv\"))\n",
+    "\n",
+    "# Generate a profile\n",
+    "profile = dp.Profiler(data)\n",
+    "\n",
```
Comment (JGSweets) [inline examples/data_profiler_demo.ipynb:453]: no need to load data and save again. otherwise LGTM

Comment (taylorfturner) [reply examples/data_profiler_demo.ipynb:453]: fine....

**Review by JGSweets — APPROVED** [review]

**Review by micdavis — APPROVED** [review]

**Pull request merged**